src/topp/MaRaClusterAdapter.cpp

```diff
@@ -366,6 +369,24 @@ class MaRaClusterAdapter :
     readMClusterOutputAsMap_(consensus_output_file, specid_to_clusterid_map, filename_to_file_idx);
     file_idx = 0;
 
+    //if specified keep original output in designated directory
+    if (!maracluster_output_directory.empty())
+    {
+      // convert path to absolute path
+      QDir mara_out_dir(maracluster_output_directory.toQString());
```

Comment:
is this conversion to absolutePath needed? 
File::copyDirRecursively in 
https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/SYSTEM/File.cpp
seems to do something similar though I don't know in detail how that works. Can you check?

Reply:
I'll check, it is the same code as in the SiriusAdapter so I guess it works.

Reply:
I guess it should not be needed any more since it should be handled by the function directly. I think this was introduced in the SiriusAdapter and then refactored into File.cpp - I guess we forgot to remove it in the SiriusAdapter.

```
    QString canonical_source_dir = source_dir.canonicalPath();
    QString canonical_target_dir = target_dir.canonicalPath();
```

Reply:
ok so when removing .absolutePath() compiling fails as it can't compile QDir to String

Reply:
Why QDir? Should work without it.

Reply:
ok, tests passed. Seems like that was used for another purpose in SiriusAdapter